src/domain/hanke/edit/HankeFormYhteystiedot.tsx

```diff
@@ -318,23 +328,22 @@ const HankeFormYhteystiedot: React.FC<FormProps> = () => {
         language={locale}
         headingLevel={3}
         heading={t('form:yhteystiedot:titles:implementerInfo')}
+        initiallyOpen={Array.isArray(toteuttajat) && toteuttajat.length > 0}
```

Comment:
Is the `Array.isArray` check needed here? Can `toteuttajat` be anything else than array?

Reply:
Probably not needed. Removed. For `rakennuttajat` too.